tests/llm/test_example_selector.py

```diff
@@ -0,0 +1,218 @@
+import asyncio
+from typing import Any, Dict, List
+
+import pytest
+import numpy as np
+from langchain_core.example_selectors.base import BaseExampleSelector
+from langchain_core.example_selectors import LengthBasedExampleSelector
+from langchain_core.prompts import FewShotPromptTemplate, PromptTemplate
+from langchain_core.messages import AIMessage, HumanMessage
+from pydantic import BaseModel, Field, RootModel
+
+from chatsky import MISC, RESPONSE
+from chatsky.llm.example_selector import Example, StaticExampleSelector, to_langchain_context
+from chatsky.responses.llm import LLMResponse
+
+np.random.seed(0)
```

Comment:
I would prefer for this to be called inside the `node_fixture` fixture (which is the only one using randomness).